secedgar/tests/utils/gen_filing_type_enum.py

```diff
@@ -0,0 +1,34 @@
+import csv
+
+def generate_signature(text, comment):
+    variable_name = text.replace('/','').replace(' ','').replace('-','').upper()
+
+    return f"FILING_{variable_name} = '{text}' # {comment}\n"
+signatures = []
+with open('tabula-edgarfm-vol2-v55.csv') as f:
```

Comment:
What is this for?

Reply:
generates all the filing types

Reply:
Let's remove this for now in favor of creating the list in the FilingType enum. Of course, we can still use this code to help us build a more extensive list of filing types.